charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -11,23 +11,23 @@ image:
 
 sidecars:
   provisionerImage:
-    repository: quay.io/k8scsi/csi-provisioner
-    tag: "v1.6.0"
+    repository: k8s.gcr.io/sig-storage/csi-provisioner
+    tag: "v2.0.2"
   attacherImage:
-    repository: quay.io/k8scsi/csi-attacher
-    tag: "v2.2.0"
+    repository: k8s.gcr.io/sig-storage/csi-attacher
+    tag: "v3.0.0"
   snapshotterImage:
-    repository: quay.io/k8scsi/csi-snapshotter
+    repository: k8s.gcr.io/sig-storage/csi-snapshotter
```

Comment:
We need at least 3.0.0 for this one it seems: https://github.com/kubernetes-csi/external-snapshotter/issues/461

Reply:
@wongma7 maybe we should get rid of the snapshotter before merging this one. Not sure which one is less effort, but this would be a throw-away effort.

Reply:
Hi Ayberk, just updated snapshotter sidecar and controller to version 3.0.3 and validated they are functioning as expected. Maybe we can retire snapshotter in a separate PR